spec/files/osws/prototype_suite.osw

```diff
@@ -1,11 +1,11 @@
 {
 	"seed_file": null,
-	"weather_file": null,
+	"weather_file": "srrl_2013_amy.epw",
```

Comment:
@macumber: Not so successful here. Noticed missing EPW messages, so attempted to simple add one - not sure if necessary.